### ISSUE TITLE
Bootstrapper and quoting of cakeArguments

### ISSUE THREAD

**Issue by Roemer** [cake-build/resources#76]
The cake arguments like `target` do not include quotes currently.
For example it looks like;
```
if ($Target) { $cakeArguments += "-target=$Target" } 
```
In (very) old versions, it did quote the variables like:
```
-target=`"$Target`" 
```
Which allowed the target to contain for example spaces. Is there any reason that the quotes were removed?

**Comment by devlead** [comment]
Boostrappers reside in resources repo, transferring this issue, would
cake-build/resources#75 solve your issue? Haven't had a chance to look at that yet.

**Comment by Roadrunner67** [comment]
@Roemer I believe this is solved in #79 and #80

**Comment by augustoproiete** [comment]
Resolved by #79, #80, and #97
